emu/src/internal_memory.rs

```diff
@@ -164,6 +197,7 @@ impl IoDevice for InternalMemory {
         match address {
             0x03000000..=0x03007FFF => self.internal_work_ram[(address - 0x03000000) as usize],
             0x04000000..=0x04000055 => self.read_address_lcd_register(address),
+            0x04000100..=0x0400010E => self.read_address_timer_register(address), // @ FB
```

Comment:
Timer registers memory goes up to `0x0400010F` since at `0x0400010E` there's `TM3CNT_H` which is 2 bytes long